docs/flowcharts/TestExecution.md

```diff
@@ -0,0 +1,45 @@
+---
+id: testexecution
+title: Test Execution
+---
+This flowchart explains the test execution process and the interaction between @wdio/runner and the most of the other WebdriverIO packages.
+<div>
+    <div class="flowcharttogglemenu">
+        <span>[</span>
+        <a class="flowcharttogglelink" onclick="createFlowChart(setupTest)">Setup test</a>
+        <span>|</span>
+        <a class="flowcharttogglelink" onclick="createFlowChart(runTest)">Run test</a>
+        <span>]</span>
+    </div>
+    <div id="flowChartGraphDiv" style="color: blue; margin-left: auto; margin-right: auto; padding-left: auto; padding-right: auto; text-align: center;"></div>
+</div>
+
+<script src="https://unpkg.com/mermaid@8.3.1/dist/mermaid.min.js"></script>
+<script src="/js/flowchart.js"></script>
+<script>
+    var setupTest = `
+    graph TD
+        START("@wdio/runner index.js called from child process via a run message.")-->
+        EXECUTERUN["Execute @wdio/runner index.js run method."]-->
+        STOREVARS["Store worker id in cid, specs in specs variable.<br>Store capabilities in caps variable."]-->
+        PARSECONFIG["Parse config file using @wdio/config configParser method.<br>Add to @wdio/runner index.js class."]-->
+        SETLOGLEVEL[Set log level]-->
+        RUNEFORESESSIONHOOK[Run wdio.conf.js beforeSession hook.]-->
+        INITSESSION[Initialise session, store in variable named browser.]-->
```

Comment:
We have some changes here now, you may see wdio runner / index.js

1. Setup Reporter
2. Init framework 
3. Before session hook
4. Init Session 
5. Before hook
6. Run tests

Reply:
Hi @mgrybyk I have updated TestExecution.md with your requested changes, please let me know if anything else needs to be changed, updated.